server/sync.go

```diff
@@ -11,9 +11,8 @@ import (
 	"github.com/mattermost/mattermost-plugin-calls/server/cluster"
 )
 
-// lockCall locks the global (cluster) mutex for the given channelID and
-// returns the current state.
-func (p *Plugin) lockCall(channelID string) (*channelState, error) {
+// lockCallSimple locks the global (cluster) mutex for the given channelID.
+func (p *Plugin) lockCallSimple(channelID string) error {
```

Comment:
I think I understand what you mean, but maybe there's a better name for this and the next? like `lockCall` and `lockCallAndReturn`? But it's a matter of taste, so up to you.

Reply:
Yeah, this is what a real `lockCall` was supposed to look like. We can rename.